src/layers/legacy/medVtkInria/vtkImageView/vtkImageView3D.cxx

```diff
@@ -667,103 +662,90 @@ void vtkImageView3D::SetTransferFunctions (vtkColorTransferFunction * color,
 //----------------------------------------------------------------------------
 void vtkImageView3D::SetOpacity (double opacity, int layer)
 {
-  if (this->HasLayer(layer))
+  if (HasLayer(layer))
   {
-    this->VolumeProperty->SetComponentWeight(layer, opacity);
-    this->GetImage3DDisplayForLayer(layer)->SetOpacity(opacity);
+    VolumeProperty->SetComponentWeight(layer, opacity);
+    GetImage3DDisplayForLayer(layer)->SetOpacity(opacity);
   }
 }
 
 //----------------------------------------------------------------------------
 double vtkImageView3D::GetOpacity(int layer) const
 {
-    double dfRes = 0.0;
-    if (this->HasLayer(layer))
-    {
-        vtkImage3DDisplay *imageDisplay = GetImage3DDisplayForLayer(layer);
-        if (imageDisplay)
-        {
-            dfRes = imageDisplay->GetOpacity();
-        }
-    }
-    return dfRes;
+    vtkImage3DDisplay * imageDisplay;
+    return (imageDisplay=GetImage3DDisplayForLayer(layer))? imageDisplay->GetOpacity() : 0.0;
```

Comment:
the previous code was more clear. Is it possible to put it back, except if there is a debug in it?